data/Store.js

```diff
@@ -50,6 +51,7 @@ export class Store {
      *      presented to loadData() prior to creating a record from that object.  This function should
      *      return a data object, taking care to clone the original object if edits are necessary.
      * @param {function} [c.filter] - filter function to be run.
+     * @param {object} [c.filterOptions] - options associated with this filter operation. See setFilter() for more details.
```

Comment:
Should we add a `@typedef` doc for these filter options since we are expecting the same object here and in setFilter?